lightning/src/util/events.rs

```diff
@@ -1090,3 +1104,49 @@ impl<T: EventHandler> EventHandler for Arc<T> {
 		self.deref().handle_event(event)
 	}
 }
+
+/// A destination that gets returned when failing to forward HTLCs
+/// This is primarily returned as a field from the `HTLCHandlingFailed` event
+#[derive(Clone, Debug, PartialEq)]
+pub enum HTLCDestination {
+	/// Represents a channel that may have been closed while we were waiting to forward to it
```

Comment:
nit: In all three variants, let's tighten these docs up by removing "Represents".